lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -323,10 +320,15 @@ PYI_CRI_ESCAPE_NO_F2F:
 
 ERROR:
   response:
-    type: error
-    pageId: pyi-technical
-    statusCode: 500
-  parent: END_JOURNEY
+    type: journey
+    journeyType: TECHNICAL_ERROR
+    initialState: ERROR
```

Comment:
We could instead make the journey transition part of the `events` configuration, so rather than defining a transient state for the journey transition we go 'directly'. e.g.:
```
SOME_STATE:
  response:
    type: page
    pageId: some-page
  events:
    error:
      targetJourney: TECHNICAL_ERROR
      targetState: ERROR
```
Might be a little tidier, but will need a bit more refactoring of the state machine